pkg/catalog/generator.go

```diff
@@ -38,6 +38,7 @@ type Generator struct {
 	FilesToCreate        map[string]string
 	MermaidFilesToCreate map[string]string
 	RedocFilesToCreate   map[string]string
+	Files                map[string][]byte
```

Comment:
Was this for reloading of template files? If so, this would be clearer as `TemplateFiles`